test/fixtures/full/main.tf

```diff
@@ -31,35 +31,35 @@ provider "gsuite" {
   version = "~> 0.1.9"
 }
 
-module "vpc" {
```

Comment:
Why are we replacing the vpc module with manually creating the subnets ourselves?

Reply:
We're doing this because the network module manages the VPC host sharing, so when the tests are torn down we disable the shared VPC project as a host. We can add a parameter making that resource optional, but for the sake of expediency I just added a couple of resources to provide the behavior we need.

Reply:
If you just leave off the shared_vpc argument from the network module, it won't affect the shared VPC status at all.